clients/lsp-ocaml.el

```diff
@@ -77,6 +77,57 @@
   :priority 0
   :server-id 'ocaml-lsp-server))
 
+(defcustom lsp-cut-signature 'space
```

Comment:
Wouldn't this change the current behavior for most lsps?
For example for clojure-lsp, server return the first line as the function signature and the rest as the docs and other things, and it's intended to be shown only the first line on messages buffer

Reply:
Since I added it to the lsp-ocaml client I thought it wouldn't have any impact on the rest of the clients. Am I wrong about it?

Reply:
oh, yes, so I think it'd be better to name it as `lsp-ocaml-cut-signature` or something